daemon/graphdriver/driver_freebsd.go

```diff
@@ -0,0 +1,8 @@
+package graphdriver
```

Comment:
Got it. Thanks for confirming. Ignore the comments then against _FreeBSD.go files.